draft-ietf-httpbis-header-structure.md

```diff
@@ -278,26 +278,19 @@ Note that commas in integers are used in this section's prose only for readabili
 
 ## Floats {#float}
 
-Floats are integers with a fractional part, that can be stored as IEEE 754 double precision numbers (binary64) ({{IEEE754}}).
+Floats are integers with a fractional part, which can be at most six digits of precision. Additionally, like integers, it can have no more than fifteen digits in total, which in some cases further constrains its precision.
```

Comment:
Might be an idea to say "decimal digits" here.

Reply:
> Floats are integers with a fractional part

I would prefere "floats are numbers with a fractional part"

Reply:
Or "numbers with integer and fractional parts"

I still think saying "decimal digits" would help, too.